src/packages.builds

```diff
@@ -28,6 +28,14 @@
     </Project>
 
     <!-- add specific builds / pkgproj's here to include in servicing builds -->
+    <Project Include="System.Private.DataContractSerialization\pkg\System.Private.DataContractSerialization.builds">
```

Comment:
You'll need to bump the version for this so that we get a new package version.  You may want to consider producing new versions of the facades on top of this, so that we have new *public* packages that folks can reference to get the fix.  Typically we don't ask folks to directly reference private packages.